plugins/woocommerce-admin/client/products/sections/images-section.scss

```diff
@@ -0,0 +1,79 @@
+.woocommerce-product-form {
+	&__images {
+		margin-bottom: 0;
+		box-shadow: unset;
+		.woocommerce-media-uploader {
```

Comment:
Good suggestion! I moved a few lines to the package in commit [a44aaa6](https://github.com/woocommerce/woocommerce/pull/34769/commits/a44aaa6469b0811ec21cd841e9b26171a95cdc3b). I left the image and the code related to that in the client because it was something specific to our implementation.